src/Multiplexer/index.js

```diff
@@ -85,17 +79,41 @@ const views = mapObjectToObject(
     ),
 );
 
+const ChromeDownloadLink = () => (
```

Comment:
```suggestion
function ChromeDownloadLink() {
```

Prefer `function` instead of lambda function